tests/operators/test_operators.py

```diff
@@ -0,0 +1,61 @@
+"""
```

Comment:
OMG tests!! \o/

We don't have CI running on this yet (see "experimental" notes in the README), but this is going to be great for development at least, thank you!

I had some comments regarding adding more tests, but I'll open an issue to add more tests.

Reply:
Awesome. Please comment away! Open to suggestions. 

Currently testing with unittest like so: `python3 -m unittest test.operators.test_operators.TestGreatExpectationsOperator`